corehq/motech/repeaters/expression/repeaters.py

```diff
@@ -25,6 +33,9 @@ class Meta:
     configured_expression = OptionValue(default=dict)
     url_template = OptionValue(default=None)
 
+    case_action_filter_expression = OptionValue(default=dict)
+    case_action_expression = OptionValue(default=dict)
```

Comment:
Aren't these specific to `CaseExpressionRepeater`? So should be added to that class?

Reply:
No, can also be a form.